float64 curvature
int32[] costs
float64 colission_r
int32[] level_inds
dwal_planner/Pose2D32[] poses
